index.html

```diff
@@ -29,61 +29,61 @@ <h1>Adopt a Dog</h1>
     <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
     <section class="dog-cards">
       <div class="card">
-        <img src="images/murphy-card.jpg" alt="Murphy">
+        <img onclick="alert('Murphy, Mutt, $123.45')" src="images/murphy-card.jpg" alt="Murphy">
         <strong>Murphy</strong>
         <span>Cost to Adopt: $123.45</span>
         <p>Corrum volorit landae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis ra ea erum, qui doluptae</p>
-        <button>Adopt</button>
+        <button onclick="cartTotal()">Adopt</button>
       </div>
       <div class="card">
-        <img src="images/poppy-card.jpg" alt="Poppy">
+        <img onclick="alert('Poppy, Terrier, $123.45')" src="images/poppy-card.jpg" alt="Poppy">
         <strong>Poppy</strong>
         <span>Cost to Adopt: $123.45</span>
         <p>Corrum volorit landae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis ra ea erum, qui doluptae</p>
-        <button>Adopt</button>
+        <button onclick="cartTotal()">Adopt</button>
       </div>
       <div class="card">
-        <img src="images/jack-card.jpg" alt="Jack">
+        <img onclick="alert('Jack, Beagle, $123.45')" src="images/jack-card.jpg" alt="Jack">
         <strong>Jack</strong>
         <span>Cost to Adopt: $123.45</span>
         <p>Corrum volorit landae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis ra ea erum, qui doluptae</p>
-        <button>Adopt</button>
+        <button onclick="cartTotal()">Adopt</button>
       </div>
       <div class="card">
-        <img src="images/duffy-card.jpg" alt="Duffy">
+        <img onclick="alert('Duffy, Mutt, $123.45')" src="images/duffy-card.jpg" alt="Duffy">
         <strong>Duffy</strong>
         <span>Cost to Adopt: $123.45</span>
         <p>Corrum volorit landae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis ra ea erum, qui doluptae</p>
-        <button>Adopt</button>
+        <button onclick="cartTotal()">Adopt</button>
```

Comment:
Same idea here, maybe think how you could pass in the cost from the HTML as a parameter and have the function take in an argument?

Reply:
I tried that but I couldn't figure out how to pull from the HTML successfully. Do you have any suggestions for where to look for that?

Reply:
Something like this....

https://codepen.io/jimmynono/pen/RmbzQb?editors=1010